internal/build/job/maker.go

```diff
@@ -70,23 +70,20 @@ func (m *maker) MakeJobTemplate(
 	owner metav1.Object,
 	pushImage bool) (*batchv1.Job, error) {
 
-	buildConfig := mld.Build
-	containerImage := mld.ContainerImage
-
 	// if build AND sign are specified, then we will build an intermediate image
 	// and let sign produce the one specified in its targetImage
+	containerImage := mld.ContainerImage
 	if module.ShouldBeSigned(mld) {
 		containerImage = module.IntermediateImageName(mld.Name, mld.Namespace, containerImage)
 	}
 
-	specTemplate := m.specTemplate(
-		mld,
-		buildConfig,
-		containerImage,
-		mld.RegistryTLS,
-		pushImage)
-
-	specTemplateHash, err := m.getHashAnnotationValue(ctx, buildConfig.DockerfileConfigMap.Name, mld.Namespace, &specTemplate)
+	specTemplate := m.specTemplate(mld, containerImage, pushImage)
+	specTemplateHash, err := m.getHashAnnotationValue(
```

Comment:
@yevgeny-shnaidman: In this case, as we are not changing the `buildConfig` before calling the `m.specTemplate`, I decided to remove it as a parameter and use the `mlm.Build` pointer. 
Is that ok, for you?

Reply:
yes, great!